src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/OffenderRepository.kt

```diff
@@ -1,11 +1,16 @@
 package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 import org.springframework.stereotype.Repository
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Offender
 
 @Repository
 interface OffenderRepository : JpaRepository<Offender, Long> {
   fun findByNomsId(nomsId: String): List<Offender>
-  fun findFirstByNomsId(nomsId: String): Offender?
+
+  @Query("select o from Offender o left join fetch o.bookings b WHERE o.nomsId = :nomsId order by b.bookingSequence asc")
+  fun findByNomsIdOrderedWithBookings(nomsId: String): List<Offender>
 }
```

Comment:
copied from prison-api since this has proved to be reliable on selecting the correct record